.github/scripts/copywrite.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+
+#
+# Checks if a file has the correct copywrite
+#
+
+year=`date +%Y`
+header="//
+// Copyright ${year} The GUAC Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the \"License\");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an \"AS IS\" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License."
+content=`cat $1`
+if  [[ $content != $header* ]]; then
+    echo $1
+fi
```

Comment:
Just a new line at the end of file missing. We should probably add '.editorconfig'to automatically solve this.